clients/admin-ui/src/features/common/nav/v2/NavSideBar.tsx

```diff
@@ -21,12 +21,17 @@ export const NavSideBar = () => {
     <VStack as="nav" align="left" spacing={4} width="200px">
       <Heading size="md">{nav.active.title}</Heading>
       <NavList>
-        {nav.active.children.map(({ title, path }) => {
+        {nav.active.children.map(({ dataTestId, path, title }) => {
           // We still need to handle cross-zone links.
           const { href, isActive } = resolveZoneLink({ href: path, router });
 
           return (
-            <NavSideBarLink key={title} href={href} isActive={isActive}>
+            <NavSideBarLink
+              dataTestId={dataTestId}
```

Comment:
I prefer not passing `dataTestId` if we can avoid it, but maybe we can't in this case? I guess because `NavLink` doesn't know which `group` it's in?

would it work to just have these have `data-testid`s of `side-nav-${title}` and update the tests accordingly?